src/SkewLinearAlgebra.jl

```diff
@@ -15,6 +15,7 @@ export
     JMatrix,
     #functions
     isskewhermitian,
+    isapproxskewhermitian,
```

Comment:
This isn't correct since `isapproxskewhermitian` is not defined in the module, it's only defined in the tests.

(It's not clear why this would be useful outside of tests for this package, anyway.)